add_to_app/multiple_flutters_module/lib/main.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+@pragma('vm:entry-point')
+void main() => runApp(MyApp(Colors.blue));
+
+@pragma('vm:entry-point')
+void topMain() => runApp(MyApp(Colors.green));
+
+@pragma('vm:entry-point')
+void bottomMain() => runApp(MyApp(Colors.purple));
+
+class MyApp extends StatelessWidget {
+  MyApp(this.color);
+
+  final Color color;
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Flutter Demo',
+      theme: ThemeData(
+        primarySwatch: this.color,
+      ),
+      home: MyHomePage(title: 'Flutter Demo Home Page'),
+    );
+  }
+}
+
+class MyHomePage extends StatefulWidget {
+  MyHomePage({Key key, this.title}) : super(key: key);
+  final String title;
+
+  @override
+  _MyHomePageState createState() => _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+  int _counter = 0;
+  MethodChannel _channel;
+
+  @override
+  void initState() {
+    super.initState();
+    _channel = MethodChannel('multiple-flutters');
+    _channel.setMethodCallHandler((MethodCall call) async {
+      if (call.method == "setCount") {
+        setState(() {
+          _counter = call.arguments as int;
+        });
+      } else {
+        throw Exception('not implemented ${call.method}');
+      }
+    });
+  }
+
+  void _incrementCounter() async {
+    _channel.invokeMethod("incrementCount", _counter);
```

Comment:
oh I see. This is a bit fragile. I'd just have the same setters on both sides whether being set locally or from the other side. Just have the same exit conditions on the setter. If being set to the same, just drop out. So no one has to worry about this implicit agreement and accidentally break it while refactoring.

Reply:
This is required if you want a single source of truth, the native datamodel.  If we removed this native views would stop updating and when you pop a view controller they would be out of date.  You'd have to create a whole new mechanism to avoid that.  If you were concerned about breaking this requirement you could make a handshake message.

Reply:
I've added a description of the data synchronization mechanism in `multiple_flutters_module/README.md`

Reply:
Ah, I see what you're trying to do. Documenting that this is just one option of doing this SG.